committee-steering/governance/sig-governance.md

```diff
@@ -82,7 +82,21 @@ Subproject contributors (as applicable).
 
 ### Chair
 
+From governance.md:
+> SIG Chairs are intended to be organizers and facilitators, responsible for 
+> the operation of the SIG and for communication and coordination with the 
+> other SIGs, the Steering Committee, and the broader community. 
+
 - Number: 2-3
+- Term: 3 years
+  - 90 days before term is up, Chair will send an email to the SIG mailing list,
```

Comment:
Normally I would be against putting work on external groups, but considering responsiveness of chairs has been an issue in the past, would it be good for this to be triggered by an external group such as Steering or SIG ContribEx/Elections Subproject to track the term start/end period and triggering this announcement?

Reply:
I'm wondering if it would make sense to also add a statement to the effect of: Chair terms should be staggered for continuity purposes

Reply:
For existing Chairs, should the 3 year term start from when they started as chair or from when the new policy takes effect? In either case, there should likely be some adjustments made for ensuring all current leads do not have their terms end at the same time.

I'm also now wondering if it would make sense to have the terms applied individually or if it would make sense to define the terms based on calendar year (and possibly staggered by SIG). Individually seems like it would be a lot harder to keep track of, where based on calendar year it could more easily be published and available for the community to see. The use of calendar year would also make it easier to leverage the [Elections Subproject](https://github.com/kubernetes/community/tree/master/elections/teams) for the voting process, though I'm not quite sure if Electo supports a differentiation between binding/non-binding votes as defined by the On-boarding and Off-barding Leads section above.

Reply:
the last comment is a good discussion topic that I was hoping to have with the wider group. 

the first comment - we can add in month and date in sigs.yaml to help track and contribex could indeed help but should be the responsibility of the sig first and foremost.

Reply:
Does this mean chairs have a 1 term limit?

Reply:
@logicalhan limits up for discussion at this point; what do you think?

Reply:
Currently worded, it almost seems implied that `term limit == 1`. I am personally for terms (as I believe in the democratic process). Term limits may end up being necessary too, but I think that (1) warrants a different thread/PR, (2) should probably be decided some time after implementing terms.

Reply:
> For existing Chairs, should the 3 year term start from when they started as chair or from when the new policy takes effect? In either case, there should likely be some adjustments made for ensuring all current leads do not have their terms end at the same time.

To strive more diversity of thought and leadership starting from the actual start date as chair would make sense. 

> limits up for discussion at this point; what do you think?

Just chiming in from my view in SIG Cluster Lifecycle, limits should be a forcing function to promote better community building, sharing of responsibilities, and generally more diversity of perspectives and thoughts. 

Limits for chairs (and perhaps TLs as well? although maybe a different discussion) can be both in terms of consecutive terms and total, an example we could have a limit of 1 consecutive term, and 5(ish?) total. There should be room for exceptions, like if nobody steps up for the role that a current chair can retain their role for a consecutive term, until a replacement is found.

Reply:
We should consider mandating the number of chairs per SIG or modifying the terms & provisions for re-staffing SIGs with only one chair. This is because, for whatever reason, if the single chair can't continue to stay on for the one-year emeritus term while finding a replacement, there is a risk associated, and we need to account for that.